JeMPI_Apps/JeMPI_UI/.env.local

```diff
@@ -1,7 +1,7 @@
 REACT_APP_JEMPI_BASE_API_HOST=http://localhost
 REACT_APP_JEMPI_BASE_API_PORT=50000
 REACT_APP_MAX_UPLOAD_CSV_SIZE_IN_MEGABYTES="128"
-
+REACT_APP_SHOW_BRAND_LOGO=false
```

Comment:
**Something to consider (not needed for now though):**  Maybe instead of passing a boolean, we should pass a string that allows use to point to another image. If it is null/undefined we default to ours, and if it is a blank string   we show nothing.

Reply:
Yes, get that...but there might be case when a particular country might want to use their own branding instead (like how google workspace allows you to add your company logo etc). Was just a thought though, so this cool. 👍🏽